packages/project/scripts/validate.ts

```diff
@@ -0,0 +1,30 @@
+import { inspect } from 'util';
+
+import { Project } from '../src/v4-specification';
+import { validateProjectData } from '../src/validation';
+import * as fs from 'fs';
+
+function validateV4(
+  projectJson: Project
+): ReturnType<typeof validateProjectData> {
+  const validatonResult = validateProjectData(projectJson);
+  return validatonResult;
+}
+
+export const main = (): void => {
+  const input = fs.readFileSync(0, 'utf-8');
```

Comment:
Whoa. This works? Everything I've seen says you should read from `process.stdin.fd`

Reply:
https://nodejs.org/api/process.html#processstdinfd

Reply:
It's certainly more reader friendly to use `process.stdin.fd`.  It was just causing typescript problems I was avoiding.

Reply:
Ah cool